Formula/qbe.rb

```diff
@@ -0,0 +1,16 @@
+class Qbe < Formula
+  desc "Compiler Backend"
+  homepage "https://c9x.me/compile/"
+  url "https://c9x.me/compile/release/qbe-1.0.tar.xz"
+  sha256 "257ef3727c462795f8e599771f18272b772beb854aacab97e0fda70c13745e0c"
+  license "MIT"
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/qbe", "-h"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Sorry, I overlooked that part in the docs. I added a basic test via 532d366.